Implemented Emojis Package and made default comments more complex

eliareutlinger opened the pull request:
Since I had errors with the current format of emojis in the comments I looked for another way to solve this. My solution is the [emoji package](https://pypi.python.org/pypi/emoji/). You can see the cheat-sheet [here](https://www.webpagefx.com/tools/emoji-cheat-sheet/).

I updated the default comments to fit the package and made them more complex, since I noticed that you get blocked quite fast if you post the same comments twice.

Would be greate if someone could check & test my changes first because i'm still a beginner in python :)

EDIT:
The comments can't be logged anymore, since the logger seems to missunderstand the format of the emojis. I weren't able to fix that yet :(
![image](https://user-images.githubusercontent.com/14804153/37899569-341b6166-30ec-11e8-8b09-c96081040a15.png)

nickpettican commented:
Looks good @eliareutlinger , have you checked the emojis appear correctly on the Instagram comments?
We should also change the README to reflect the changes. I'll test it out when I get some free time this week :), and also fix the logging issue.

eliareutlinger commented:
@nickpettican jep, they appear correctly:
![image](https://user-images.githubusercontent.com/14804153/37902980-89480f04-30f6-11e8-8b0f-d80d5e38a173.png)
I know the complexity makes it partly ugly but i'm runnig it now since one hour and didn't got blocked yet :)

eliareutlinger commented:
Well, not all emojis from the given cheat-sheet seem to work.
For Example ":simple_smile:" wasn't even translated by the emoji-pack. It just says :simple_smile: in the comment. So I replaced it with :smiley: which is working properly. 

So idk which ones are actually working fine, but the ones in the default config do. Seems like you'd have to check others first before using them.

eliareutlinger commented:
I think I fixed the Loggin now (by printing the emojis in their alphabetic code first) and added some more comments. I think this should be working but since Instagram seems to have a big problem with their Login-UI I weren't able to test it yet 👎

eliareutlinger commented:
The emojis are now working fine. I tested everything this day for about 8 Hours on Raspbian and had no single Error logged. Additionally I changed the console output/logging structure to display informations clearer.

nickpettican commented on config.json:
```diff
@@ -11,10 +11,61 @@
-	"comments_list": [["Cool", "Sweet", "Awesome", "Great"],
-						["😄", "🙌", "👍", "👌", "😊"], 
-						[".", "!", "!!", "!!!"]],
+	"comments_in_day": 50,
```
Good work here

nickpettican commented on src/instabot.py:
```diff
@@ -300,15 +301,15 @@ def catch_up_operations(self):
             self.banned['banned'] = True
             self.banned['400'] = 0
-        self.console.log('\tSleeping for %s minutes... '%(sleep_time/60))
+        self.console.log('\n\t--- Sleeping for %s minutes ---'%(sleep_time/60))
```
Better logging indeed

eliareutlinger replied:
Not better, but clearer in console (for me at least)

nickpettican commented on src/instabot.py:
```diff
@@ -685,31 +687,28 @@ def unfollow_user(self):
     def user_following_back(self, username):
         # check if the user is following back
 
-        user_data = check_user(self.browser, self.insta_urls['user'], username)
```
Why was this removed?

eliareutlinger commented:
I removed the function for betterlocal testing (on my windows machine), since I didn't want to follow people with a test account which is never active. 

Then I forgot to redo it, but here you go

nickpettican commented on src/instabot.py:
```diff
@@ -754,8 +756,9 @@ def explore_operation(self, operation, identifier):
-                comment = generate_comment(self.params['comments_list'])
-                self.console.log(comment + '\,')
+                commentString = generate_comment(self.params['comments_list'])
+                self.console.log('\n\n\tComment: "' + commentString + '" \n')
```
Should we remove the last `\n` and add `\,` to have `Success` printed in the same line? Otherwise it looks like this:
```
[ 2018-03-27 16:11:55 ]  * Trying to comment "https://www.instagram.com/p/":  

	Comment: "Cool :star: :clap: Keep it up!" 

Success
```

nickpettican replied:
Other than that it works fine 👍 I'll merge once you reply to this ☝️ @eliareutlinger

eliareutlinger replied:
I didn't really knew where to should put that `Success` because of the long comments. But yes that'd look better.
